pkg/config/config.go

```diff
@@ -509,17 +511,17 @@ func (c *Config) BlockstoreAzureParams() (blockparams.Azure, error) {
 		logging.ContextUnavailable().Warn("blockstore.azure.auth_method is deprecated. Value is no longer used.")
 	}
 	if c.Blockstore.Azure.ChinaCloud {
-		logging.ContextUnavailable().Warn("blockstore.azure.china_cloud is enabled. lakeFS will only function on Azure China Cloud")
+		logging.ContextUnavailable().Warn("blockstore.azure.china_cloud is deprecated. Value is no longer used.")
```

Comment:
Shouldn't we exit? If someone is using this, by design, lakeFS won't work for him.

Reply:
I think we should actually do something else so it wouldn't be a breaking change - fixing